scripts/mkcloud

```diff
@@ -1455,6 +1458,14 @@ function sanity_checks
             ;;
     esac
 
+    case $monasca_tsdb in
+        ''|influxdb|cassandra)
+            ;;
+        *)
+            complain 101 "$monasca_tsdb as the Monasca time-series DB is currently not supported."
```

Comment:
This validation should be done in crowbar imho (as a schema validator)

Reply:
Removed here and added in [Crowbar](https://github.com/crowbar/crowbar-openstack/pull/2061/files#diff-c13d54db5e9dfd8275021e09bbf8bc6eR237).